packages/orchestration/test/examples/snapshots/unbond.contract.test.ts.md

```diff
@@ -28,6 +28,18 @@ Generated by [AVA](https://avajs.dev).
         orchestration: {
           unbondAndLiquidStake: {
             asyncFlow_kindHandle: 'Alleged: kind',
+            endowments: {
+              0: {
+                zcfTools: {
+                  assertUniqueKeyword_kindHandle: 'Alleged: kind',
+                  assertUniqueKeyword_singleton: 'Alleged: assertUniqueKeyword',
+                  atomicRearrange_kindHandle: 'Alleged: kind',
+                  atomicRearrange_singleton: 'Alleged: atomicRearrange',
+                  makeInvitation_kindHandle: 'Alleged: kind',
+                  makeInvitation_singleton: 'Alleged: makeInvitation',
+                },
```

Comment:
Yeah I'm still not sure if it might not be better to have a single exo here.

Reply:
does that go for zoeTools too?

Reply:
I only see `localTransfer` being explicitly picked in examples. I suppose that's an argument for keeping as-is: one might expect to use these tools in isolation without passing the whole "tools" object.